ui/apps/platform/src/Containers/ComplianceEnhanced/Dashboard/ScanResultsOverviewTable.tsx

```diff
@@ -0,0 +1,100 @@
+import {
+    TableComposable,
+    Tbody,
+    Td,
+    Th,
+    Thead,
+    Tr,
+    ActionsColumn,
+    IAction,
+} from '@patternfly/react-table';
+import { format } from 'date-fns';
+
+import usePagination from 'hooks/patternfly/usePagination';
+import useTableSort from 'hooks/patternfly/useTableSort';
+
+import { ComplianceScanResultsOverview } from './types';
+import ScanResultsToolbar, { SearchFilter } from './ScanResultsToolbar';
+
+import scanResultsOverviewMockData from './MockData/complianceScanResultsOverview.json';
+
+const defaultSearchFilter = {
+    scanName: '',
+    clusterName: '',
+    profileName: '',
+};
+
+const sortFields = ['Scan Name', 'Failing Controls', 'Last Scanned'];
+const defaultSortOption = { field: 'Last Scanned', direction: 'asc' } as const;
+
+function ScanResultsOverviewTable() {
+    const { page, perPage, onSetPage, onPerPageSelect } = usePagination();
+    const { sortOption, getSortParams } = useTableSort({ sortFields, defaultSortOption });
+    const [searchFilter, setSearchFilter] = useState<SearchFilter>(defaultSearchFilter);
+    const [scanResultsOverviewData] = useState<ComplianceScanResultsOverview[]>(
+        scanResultsOverviewMockData.scanOverviews
+    );
+
+    useEffect(() => {
+        // TODO: fetch GetComplianceScanResultsOverview once implemented and update scanResultsOverviewData
+    }, [searchFilter, page, perPage, sortOption]);
+
+    const defaultActions = (): IAction[] => [
+        {
+            title: 'Edit schedule',
+            // TODO: link to edit schedule page
+        },
+    ];
+
+    const displayOnlyItemOrItemCount = (items: string[], multipleItemLabel: string): string => {
+        if (items.length > 1) {
+            return `${items.length} ${multipleItemLabel}`;
+        }
+        return items[0];
+    };
+
+    return (
+        <>
+            <ScanResultsToolbar
+                numberOfScanResults={scanResultsOverviewData.length}
+                searchFilter={searchFilter}
+                setSearchFilter={setSearchFilter}
+                page={page}
+                perPage={perPage}
+                onSetPage={onSetPage}
+                onPerPageSelect={onPerPageSelect}
+            />
+            <TableComposable borders={false}>
+                <Thead noWrap>
+                    <Tr>
+                        <Th sort={getSortParams('Scan Name')}>Scan</Th>
+                        <Th>Clusters</Th>
+                        <Th>Profiles</Th>
+                        <Th sort={getSortParams('Failing Controls')}>Failing Controls</Th>
+                        <Th sort={getSortParams('Last Scanned')}>Last Scanned</Th>
+                        <Td />
+                    </Tr>
+                </Thead>
+                <Tbody>
+                    {scanResultsOverviewData.map(({ scanStats, clusterId, profileName }) => {
+                        return (
+                            <Tr key={scanStats.scanName}>
```

Comment:
I know the API data is TBD, but it might be worth a comment here to change the key if `scanName` ends up being something that isn't unique.